pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -45,6 +48,42 @@ func (ms *MetricSender) ReportNetworkDeviceMetadata(config *checkconfig.CheckCon
 		}
 		ms.sender.EventPlatformEvent(string(payloadBytes), epforwarder.EventTypeNetworkDevicesMetadata)
 	}
+
+	// Telemetry
+	for _, interfaceStatus := range interfaces {
+		status := computeInterfaceStatus(interfaceStatus.AdminStatus, interfaceStatus.OperStatus)
+		interfaceTags := []string{"status:" + status, "interface:" + interfaceStatus.Name, "interface_alias:" + interfaceStatus.Alias}
+		interfaceTags = append(interfaceTags, tags...)
+		ms.sender.Gauge(interfaceStatusMetric, 1, "", interfaceTags)
+	}
+}
+
+func computeInterfaceStatus(adminStatus int32, operStatus int32) string {
+	if adminStatus == 1 {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think it can add readability to define constants for adminStatus/operStatus and then being able to write conditions like:

```
if adminStatus == adminStatusUp {
```